jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditorDialog.kt

```diff
@@ -49,8 +51,12 @@ class QueryEditorDialog(
     }
 
     private fun setView(queryDetails: QueryDetails) {
-        // TODO: handle multiple groups
-        view.logGroupLabel.text = "Log Group : ${queryDetails.logGroups.first()}"
+        val availableLogGroups = AwsResourceCache.getInstance(project).getResourceNow(
```

Comment:
This is done on the UI thread so if we don't have it (which is technically possible) this will make a network call on the UI thread